### PR TITLE
NDEV-2807: Update Solana to 1.17.28

### PR DESCRIPTION
**Pull request by stanislav-tkach**

### CODE REVIEW

File: evm_loader/Cargo.toml [hunk 1]
```diff
@@ -10,3 +10,13 @@ members = [
     'program',
     'program-macro',
 ]
+
+[workspace.dependencies]
+solana-clap-utils = "=1.17.28"
```
Comment (andreisilviudragnea) [inline evm_loader/Cargo.toml:15]: The PR misses changes in Cargo.lock file

**Comment by neonlabstech** [comment]
<details><summary>Dapps report</summary>

File: evm_loader/Cargo.toml (hunk 1 above)
Comment (stanislav-tkach) [reply evm_loader/Cargo.toml:15]: Fixed.

**Review by andreisilviudragnea — APPROVED** [review]

**Review by andreisilviudragnea — APPROVED** [review]

**Review by Deniskore — APPROVED** [review]

**Comment by neonlabstech** [comment]
<details><summary>Dapps report</summary>

**Pull request merged**